nf_core/create.py

```diff
@@ -88,77 +237,199 @@ def render_template(self):
+
+                # Copy the file directly instead of using Jinja
+                except (AttributeError, UnicodeDecodeError) as e:
+                    log.debug(f"Copying file without Jinja: '{output_path}' - {e}")
+                    shutil.copy(template_fn_path, output_path)
+
+                # Something else went wrong
+                except Exception as e:
+                    log.error(f"Copying raw file as error rendering with Jinja: '{output_path}' - {e}")
+                    shutil.copy(template_fn_path, output_path)
+
+                # Mirror file permissions
+                template_stat = os.stat(template_fn_path)
+                os.chmod(output_path, template_stat.st_mode)
+
+        # Remove all unused parameters in the nextflow schema
+        if not self.template_params["igenomes"] or not self.template_params["nf_core_configs"]:
+            self.update_nextflow_schema()
+
+        if self.template_params["branded"]:
+            # Make a logo and save it, if it is a nf-core pipeline
+            self.make_pipeline_logo()
+        else:
+            # Remove field mentioning nf-core docs
+            # in the github bug report template
+            self.remove_nf_core_in_bug_report_template()
+
+            # Update the .nf-core.yml with linting configurations
+            self.fix_linting()
+
+    def update_nextflow_schema(self):
+        """
+        Removes unused parameters from the nextflow schema.
+        """
+        schema_path = os.path.join(self.outdir, "nextflow_schema.json")
+
+        schema = nf_core.schema.PipelineSchema()
+        schema.schema_filename = schema_path
+        schema.no_prompts = True
+        schema.load_schema()
+        schema.get_wf_params()
+        schema.remove_schema_notfound_configs()
+        schema.save_schema(suppress_logging=True)
+
+        # The schema is not guaranteed to follow Prettier standards
+        # so we run prettier on the schema file
+        subprocess.run(["prettier", "--write", schema_path], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+
+    def remove_nf_core_in_bug_report_template(self):
+        """
+        Remove the field mentioning nf-core documentation
+        in the github bug report template
+        """
+        bug_report_path = os.path.join(self.outdir, ".github", "ISSUE_TEMPLATE", "bug_report.yml")
+
+        with open(bug_report_path, "r") as fh:
+            contents = yaml.load(fh, Loader=yaml.FullLoader)
+
+        # Remove the first item in the body, which is the information about the docs
+        contents["body"].pop(0)
+
+        with open(bug_report_path, "w") as fh:
+            yaml.dump(contents, fh, default_flow_style=False, sort_keys=False)
+
+        # The dumped yaml file will not follow prettier formatting rules
+        # so we run prettier on the file
+        subprocess.run(["prettier", "--write", bug_report_path], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
This fails with `FileNotFoundError: [Errno 2] No such file or directory: 'prettier'` when prettier is not installed, so we could add a better error message as discussed with @ErikDanielsson